src/api/environment.cc

```diff
@@ -512,8 +512,16 @@ NODE_EXTERN std::unique_ptr<InspectorParentHandle> GetInspectorParentHandle(
 
 NODE_EXTERN std::unique_ptr<InspectorParentHandle> GetInspectorParentHandle(
     Environment* env, ThreadId thread_id, const char* url, const char* name) {
+  std::string name_str(name);
+  return GetInspectorParentHandle(env, thread_id, url, name_str);
+}
+
+NODE_EXTERN std::unique_ptr<InspectorParentHandle> GetInspectorParentHandle(
+    Environment* env,
+    ThreadId thread_id,
+    const char* url,
+    const std::string& name) {
   CHECK_NOT_NULL(env);
-  if (name == nullptr) name = "";
```

Comment:
This check would need to be moved to the wrapper function

Reply:
Yes, the wrapper function still takes a `const char*` and correspondingly does need this check

Reply:
Oh! I get it.